docs/release-procedure.md

```diff
@@ -0,0 +1,15 @@
+# Release procedure
```

Comment:
link to this in the contribution guide please

Reply:
Update in https://github.com/WildMeOrg/codex-frontend/pull/424/commits/5f52be6481f1d4a5568fa496ff399b5d68ad7346